runtime/wasm/src/host.rs

```diff
@@ -774,17 +774,20 @@ impl RuntimeHostTrait for RuntimeHost {
             })
             .collect::<Vec<_>>();
 
-        // If none of the handlers match the event params, fail processing the event
+        // If none of the handlers match the event params, log a warning and
+        // skip the event. See https://github.com/graphprotocol/graph-node/issues/1021
```

Comment:
I'm fine with referencing issues with just `#1021` if you also prefer that.

Reply:
I am :)